MOCC/src/configurations/NewPyTango/device_proxy.py

```diff
@@ -0,0 +1,38 @@
+import requests
+
+
+class DeviceProxy:
+    def __init__(self, resource_path_URI):
+        self.resource_path_URI = resource_path_URI
+        self.bind_functions()
+
+    def test_connection(self):
+        return 'SUCCESS'
+
+    # Getting the functions of the device server
+    def get_function_list(self):
+        uri = 'http://127.0.0.1:5000' + self.resource_path_URI + '/functions'
```

Comment:
the resource_path_uri has the format **/root/system/subsystem/service/component/instance** (some parts might be missing)
That should be a key in the database, which you should look up to get the IP and port of the server providing the **instance** functionality.
For example, **/c3/health_monitor/1** maps to **http://127.0.0.1:5000**, and the mapping is stored in the database.